blueprints/cloud-operations/network-dashboard/variables.tf

```diff
@@ -50,28 +50,35 @@ variable "organization_id" {
 
 variable "prefix" {
   description = "Customer name to use as prefix for monitoring project"
+  default     = "mon"
 }
 
+
 variable "project_monitoring_services" {
   description = "Service APIs enabled in the monitoring project if it will be created."
   default = [
     "artifactregistry.googleapis.com",
     "cloudasset.googleapis.com",
     "cloudbilling.googleapis.com",
     "cloudbuild.googleapis.com",
+    "cloudfunctions.googleapis.com",
     "cloudresourcemanager.googleapis.com",
     "cloudscheduler.googleapis.com",
     "compute.googleapis.com",
-    "cloudfunctions.googleapis.com",
     "iam.googleapis.com",
     "iamcredentials.googleapis.com",
     "logging.googleapis.com",
     "monitoring.googleapis.com",
+    "pubsub.googleapis.com",
     "run.googleapis.com",
-    "serviceusage.googleapis.com"
+    "servicenetworking.googleapis.com",
+    "serviceusage.googleapis.com",
+    "storage-component.googleapis.com"
   ]
 }
 
+
+
```

Comment:
There's a few new empty lines in the file. Can you keep a single empty line between variables?

Reply:
kk